pkg/ebpf/tracee.go

```diff
@@ -1370,6 +1356,13 @@ func (t *Tracee) Run(ctx gocontext.Context) error {
 	t.bpfLogsPerfMap.Start()
 	go t.processBPFLogs()
 	t.running = true
+
+	// write pid file
+	err = t.writePid()
+	if err != nil {
+		logger.Warn("memory dump", "error", err)
```

Comment:
Error message seems wrong? Also I think we should log error and return here (with `t.Close()` called), not continue without the pid file.

Reply:
Yep, we need to abort.

Reply:
If CNDR depends on it, probably best to abort in case it was not able to write?